runtime/src/types.rs

```diff
@@ -56,7 +56,10 @@ pub enum Body {
     },
 
     // Runtime worker interface.
-    WorkerInfoRequest {},
+    WorkerInfoRequest {
+        #[serde(with = "serde_bytes")]
+        runtime_id: Vec<u8>,
```

Comment:
Why not use the `RuntimeId` type?

Reply:
Will it import/export itself nicely with `serde_cbor`, if I use `RuntimeId` type?

Reply:
Both should work.